CODEOWNERS

```
@@ -0,0 +1,11 @@
+# To be kept in sync with: [community/OWNERS](https://github.com/veraison/community/blob/main/OWNERS) 
+# and the GitHub Team: [go-cose-maintainers](https://github.com/orgs/veraison/teams/go-cose-maintainers)
+
+henkbirkholz
+qmuntal
+setrofim
+shizhMSFT
+simonfrost-arm
+SteveLasker
+thomas-fossati
+yogeshbdeshpande
```

According to [this doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), this file should be formatted as
```suggestion
* @henkbirkholz @qmuntal @setrofim @shizhMSFT @simonfrost-arm @SteveLasker @thomas-fossati @yogeshbdeshpande
```